snaps_tests/__main__.py

```diff
@@ -47,7 +48,11 @@ def main():
 
     arguments = docopt.docopt(__doc__)
 
-    snaps_tests.config['skip-install'] = arguments['--skip-install']
+    if platform.machine() == 'armv7l':
```

Comment:
Or don't use project_options if you are really against it, but set it up only once in `setUp`